### ISSUE TITLE
#30 broke OSX builds

### ISSUE THREAD

**Issue by pshipton** [ibmruntimes/openj9-openjdk-jdk12#34]
#30 was reverted and needs to be re-instated with a fix for this osx build problem.

```
17:49:45  /Users/jenkins/workspace/PullRequest-Compile-JDK12-osx_x86-64_cmprssptrs-OpenJDK12/src/java.desktop/macosx/classes/sun/lwawt/macosx/CWarningWindow.java:72: error: cannot find symbol
17:49:45                  icons[0][0] = new IconInfo(sun.awt.AWTIcon32_security_icon_bw16_png.security_icon_bw16_png);
17:49:45                                                    ^
17:49:45    symbol:   class AWTIcon32_security_icon_bw16_png
17:49:45    location: package sun.awt
17:49:45  /Users/jenkins/workspace/PullRequest-Compile-JDK12-osx_x86-64_cmprssptrs-OpenJDK12/src/java.desktop/macosx/classes/sun/lwawt/macosx/CWarningWindow.java:73: error: cannot find symbol
```

**Comment by pshipton** [comment]
@keithc-ca

**Comment by keithc-ca** [comment]
This is due to ibmruntimes/openj9-openjdk-jdk10#17 which broke the discovery of dependencies among modules.

**Comment by keithc-ca** [comment]
The underlying issue was fixed via eclipse/openj9#5260.
